pandas/plotting/_matplotlib/boxplot.py

```diff
@@ -296,10 +297,18 @@ def maybe_color_bp(bp, **kwds):
         if not kwds.get("capprops"):
             setp(bp["caps"], color=colors[3], alpha=1)
 
-    def plot_group(keys, values, ax: "Axes"):
+    def plot_group(keys, values, ax: "Axes", plot_points: bool = False):
         keys = [pprint_thing(x) for x in keys]
         values = [np.asarray(remove_na_arraylike(v), dtype=object) for v in values]
         bp = ax.boxplot(values, **kwds)
+
+        if plot_points:
+            # if plot_points
+            for x, ys in enumerate(values):
+                ax.plot(
+                    x + np.random.normal(size=len(ys)) * 0.04 + 1, ys, "ro", alpha=0.5
```

Comment:
Can you explain what this 0.04 and + 1 mean here?
I suggest to extract a function and use named variables there for clarity.